runtime/syntax/logtalk.vim

```diff
@@ -386,47 +408,59 @@ syn sync ccomment logtalkBlockComment maxlines=50
 
 
 " Define the default highlighting.
-" Only when an item doesn't have highlighting yet
+" For version 5.7 and earlier: only when not done already
+" For version 5.8 and later: only when an item doesn't have highlighting yet
```

Comment:
Same here. We don't need to care pre-6.0.
We should use `hi def link` directly.

Reply:
Logtalk is a +20 years old programming language and Vim support was added indeed long time ago :-)

Reply:
See this commit: f37506f60f87d52a9e8850e30067645e2b13783c
All `HiLink` commands were removed from all syntax files including logtalk.
Even the support was added long time ago, Vim is still evolving ;-)

Reply:
How best to proceed? Bram already closed the pull request. Should I leave to him or other maintainer to remove those `HiLink` commands or should I submit a new pull request?

Reply:
I think we can leave to Bram.
If the files are not properly merged after next runtime files update, please create a new PR then.

Reply:
👍